odl/phantom/transmission.py

```diff
@@ -141,6 +144,183 @@ def shepp_logan(space, modified=False):
+         [-4.3, 6.8, 1.8, 0.24, -30, 0.750, 0],  # 10
+         [4.3, 6.8, 1.8, 0.24, 30, 0.750, 0],  # 11
+         [0, -3.6, 1.8, 3.6, 0, -0.005, 0],  # 12
+         [6.39395, -6.39395, 1.2, 0.42, 58.1, 0.005, 0],  # 13
+         [0, 3.6, 2, 2, 0, 0.750, 4],  # 14
+         [0, 9.6, 1.8, 3.0, 0, 1.800, 4],  # 15
+         [0, 0, 9.0, 11.4, 0, 0.750, 3],  # 16a
+         [0, y016b, a16b, b16b, 0, 0.750, 1],  # 16b
+         [0, 0, 9.0, 11.4, 0, -0.750, ear],  # 6
+         [9.1, 0, 4.2, 1.8, 0, 0.750, 1]]  # R_ear
+    E = np.array(E)
+
+    # generate the air cavities in the right ear
+    cavity1 = np.arange(8.8, 5.6, -0.4)[:, None]
+    cavity2 = np.zeros([9, 1])
+    cavity3_7 = np.ones([53, 1]) * [0.15, 0.15, 0, -1.800, 0]
+
+    for j in range(1, 4):
+        kj = 8 - 2 * int(np.floor(j / 3))
+        dj = 0.2 * int(np.mod(j, 2))
+
+        cavity1 = np.vstack((cavity1,
+                             cavity1[0:kj] - dj,
+                             cavity1[0:kj] - dj))
+        cavity2 = np.vstack((cavity2,
+                             j * sha * np.ones([kj, 1]),
+                             -j * sha * np.ones([kj, 1])))
+
+    E_cavity = np.hstack((cavity1, cavity2, cavity3_7))
+
+    # generate the left ear (resolution pattern)
+    x0 = -7.0
+    y0 = -1.0
+    d0_xy = 0.04
+
+    d_xy = [0.0357, 0.0312, 0.0278, 0.0250]
+    ab = 0.5 * np.ones([5, 1]) * d_xy
+    ab = ab.T.ravel()[:, None] * np.ones([1, 4])
+    abr = ab.T.ravel()[:, None]
+
+    leftear4_7 = np.hstack([abr, abr, np.ones([80, 1]) * [0, 0.75, 0]])
+
+    x00 = np.zeros([0, 1])
+    y00 = np.zeros([0, 1])
+    for i in range(1, 5):
+        y00 = np.vstack((y00,
+                         (y0 + np.arange(0, 5) * 2 * d_xy[i - 1])[:, None]))
+        x00 = np.vstack((x00,
+                         (x0 + 2 * (i - 1) * d0_xy) * np.ones([5, 1])))
+
+    x00 = x00 * np.ones([1, 4])
+    x00 = x00.T.ravel()[:, None]
+    y00 = np.vstack([y00, y00 + 12 * d0_xy,
+                     y00 + 24 * d0_xy, y00 + 36 * d0_xy])
+
+    leftear = np.hstack([x00, y00, leftear4_7])
+    C = [[1.2, 1.2, 0.27884, 0.27884, 0.60687, 0.60687, 0.2,
+          0.2, -2.605, -2.605, -10.71177, y016b + 10.71177, 8.88740, -0.21260],
+         [0, 180, 90, 270, 90, 270, 0,
+          180, 15, 165, 90, 270, 0, 0]]
+    C = np.array(C)
+
+    if not resolution and not ear:
+        phantomE = E[:17, :]
+        phantomC = C[:, :12]
+    elif not resolution and ear:
+        phantomE = np.vstack([E, E_cavity])
+        phantomC = C
+    elif resolution and not ear:
+        phantomE = np.vstack([leftear, E[:17, :]])
+        phantomC = C[:, :12]
+    else:
+        phantomE = np.vstack([leftear, E, E_cavity])
+        phantomC = C
+
+    return phantomE, phantomC
+
+
+def forbild(space, resolution=False, ear=True):
+    """Standard `FORBILD phantom` in 2 dimensions.
+
+    The forbild phantom is intended for testing CT-algorithms and is intended
+    to be similar to a human head.
```

Comment:
I don't know.. the resolution pattern doesn't look very much like anything in my head.